src/IO/Observer/Helpers.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "IO/Observer/Tags.hpp"
+#include "Parallel/Reduction.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TypeTraits.hpp"
+
+namespace observers {
+namespace detail {
+template <class Observer, class = cpp17::void_t<>>
+struct get_reduction_data_tags_from_observer {
+  using type = tmpl::list<>;
+};
+
+template <class Observer>
+struct get_reduction_data_tags_from_observer<
+    Observer, cpp17::void_t<typename Observer::reduction_data_tags>> {
+  using type = typename Observer::reduction_data_tags;
+};
+}  // namespace detail
+
+/// Each Observer should specify a type alias `reduction_data_tags`.
```

Comment:
Users don't write observers so this is using nomenclature incorrectly. I think what you mean is `Each action that sends data to the observers...` and is it optional to specify the type alias or required? If required I'd suggest `should` -> `must`. Also, if not required would that mean that you can have an action that doesn't observe anything? I'm confused by what the intended use case is there.

Reply:
In the fixup I just pushed, I changed the wording to "must".

However, if for some reason someone wants to specify `Metavariables::reduction_data_tags` by hand instead of using `get_reduction_data_tags_from_observing_actions` (this used to be done in EvolveValenciaDivClean.hpp and was commented with "Hack" because it requires EvolveValenciaDivClean to know exactly what every observing action does), then there is no requirement for observing actions to specify `reduction_data_tags`.